CC0LRB/Todo/Models/ViewModels/TodoViewModel.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace Todo.Models.ViewModels
+{
+    public class TodoViewModel
+    {
+        public List<TodoItem> TodoList { get; set; }
+        public TodoItem Todo { get; set; }
```

Comment:
Itt a Todo gondolom az aktuális elemet jelenti. Ebben az esetben lehet találóbb lett volna az Actual név, illetve a TodoList helyett találóbb név lenne a Todos. A típus nevet azért nem szerencsés reflektálni a változónévben, mert ha változik a típus, akkor a nevet is illik utánahúzni. Ennek elmaradása csak később félreértéseket okozhat.